### PR TITLE
Support uint16 color layers

### LINKED ISSUE #3781: Support for uint16 color data
Many microscopes, CTs etc output 16 or even 32 Bit data. WKW also supports these formats. Would be great if wk could also display this data. I think the data would be recalculated to 8-Bit in a shader with contrast/brightness settings in mind.

### PR DESCRIPTION
**Pull request by daniel-wer**
This PR adds uint16 color layer support. Histograms are not working correctly yet, very related to https://github.com/scalableminds/webknossos/issues/4095.

Backend ToDo (highest priority first):
- [x] ~~the backend doesn't send enough data for uint16 color layer buckets, looks like it automatically assumes uint8~~
- [ ] Histogram data for uint16 color layers is not correct yet. Can probably be done as part of #4095.
- [x] ~~the backend doesn't detect uint16 color layers correctly when importing the dataset (detects uint8 instead)~~

### URL of deployed dev instance (used for testing):
- https://supportuintcolor.webknossos.xyz

### Steps to test:
- Download the data_types dataset from our wiki which contains a uint16 layer.

### Issues:
- fixes #3781 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [x] Updated [documentation](../blob/master/docs) if applicable
- [x] Ready for review

### CODE REVIEW

**Comment by daniel-wer** [comment]
I think I'll disable the histogram for now :)

**Comment by daniel-wer** [comment]
I've disable the histogram for all layers but (u)int8 for now. Although it was enabled for uint24 layers before, what was displayed was definitely not correct.

File: frontend/javascripts/oxalis/view/settings/dataset_settings_view.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
     const { alpha, color, intensityRange, isDisabled } = layer;
     let histogram = new Array(256).fill(0);
     if (this.state.histograms && this.state.histograms[layerName]) {
-      histogram = this.state.histograms[layerName].histogram;
+      ({ histogram } = this.state.histograms[layerName]);
```
Comment (philippotto) [inline frontend/javascripts/oxalis/view/settings/dataset_settings_view.js]: Interesting, didn't know that syntax for already defined variables!

File: frontend/javascripts/oxalis/view/settings/histogram_view.js [hunk 2]
```diff
@@ -27,6 +28,13 @@ type HistogramProps = {
 const canvasHeight = 100;
 const canvasWidth = 300;
 
+export function isHistogramSupported(elementClass: ElementClass): boolean {
+  if (["int8", "uint8"].includes(elementClass)) {
```
Comment (philippotto) [inline frontend/javascripts/oxalis/view/settings/histogram_view.js]: How about `return [...].includes(...)`? :)

Comment (daniel-wer) [reply frontend/javascripts/oxalis/view/settings/histogram_view.js]: :see_no_evil:

**Pull request merged**